libraries/apollo-mockserver/api/apollo-mockserver.api

```diff
@@ -31,33 +45,25 @@ public final class com/apollographql/apollo3/mockserver/MockResponse$Builder {
 	public final fun statusCode (I)Lcom/apollographql/apollo3/mockserver/MockResponse$Builder;
 }
 
-public final class com/apollographql/apollo3/mockserver/MockServer : com/apollographql/apollo3/mockserver/MockServerInterface {
```

Comment:
`MockServerInterface` is no more. There is no more `expect class` because of [this tweet](https://twitter.com/Sellmair/status/1647932805647572994). There is only a `expect fun MockServer()`